packages/flutter_test/lib/src/window.dart

```diff
@@ -347,6 +347,11 @@ class TestWindow implements Window {
     _window.onPlatformMessage = callback;
   }
 
+  @override
+  dynamic noSuchMethod(Invocation invocation) {
```

Comment:
What does this change do?

Reply:
Oh my bad!  I was testing some things, there are still some incompatibilities with the web ui